### PR TITLE
Add Quicksight docs

### PR DESCRIPTION
**Pull request by ayush-shah**
### Describe your changes:

Add Quicksight docs

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

**Review by NiharDoshi99 — DISMISSED** [review]

File: openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/airflow.md [hunk 1]
```diff
@@ -18,6 +18,55 @@ Configure and schedule Quicksight metadata and profiler workflows from the OpenM
 To deploy OpenMetadata, check the Deployment guides.
 {%/inlineCallout%}
 
+AWS Quicksight Permissions
```
Comment (nahuelverdugo) [inline openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/airflow.md:21]:
```suggestion
### AWS Quicksight Permissions
```

wdyt?

Comment (nahuelverdugo) [reply openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/airflow.md:21]: Same comment for other places where we have the exact text.

File: openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/index.md [hunk 2]
```diff
@@ -185,7 +218,7 @@ desired.
   - **Endpoint URL (optional)**: Your Glue connector will automatically determine the AWS Glue endpoint URL based on the region. You may override this behavior by entering a value to the endpoint URL.
```
Comment (nahuelverdugo) [inline openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/index.md:218]: I know this is not part of your change but why it says Glue connector?

**Review by pmbrull — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md [hunk 3]
```diff
@@ -16,80 +64,132 @@ AWS credentials configs.
...
-The Amazon Resource Name (ARN) of the role to assume. Optional Field in case of Assume Role
-<!-- assumeRoleSourceIdentity to be updated -->
+The source identity specified by the principal that is calling the `AssumeRole` operation. You can use source identity
+information in AWS CloudTrail logs to determine who took actions with a role.
+
+Find more information about [Source Identity](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html#:~:text=Required%3A%20No-,SourceIdentity,-The%20source%20identity).
 $$
 
 $$section
 ### Aws Account Id $(id="awsAccountId")
```
Comment (nahuelverdugo) [inline openmetadata-ui/src/main/resources/ui/public/locales/en-US/Dashboard/QuickSight.md:166]:
```suggestion
### AWS Account Id $(id="awsAccountId")
```

Let's review where else we are using `Aws` or `aws` and change it with `AWS`.

File: openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/airflow.md [hunk 4]
```diff
@@ -18,6 +18,55 @@ Configure and schedule Quicksight metadata and profiler workflows from the OpenM
...
+| 3 | ListDataSources |
+| 4 | ListDashboards |
+| 5 | DescribeAnalysis |
+| 6 | DescribeDataSet |
+| 7 | ListDataSets |
+| 8 | DescribeDataSource |
+
+Here is how to add Permissions to an IAM user.
+
+- Navigate to the IAM console in the AWS Management Console.
```
[the remark this answers is not in the document]
Comment (nahuelverdugo) [reply openmetadata-docs-v1/content/v0.13.2/connectors/dashboard/quicksight/airflow.md:37]: Same comment for other places where we have the exact text.

**Review by nahuelverdugo — CHANGES_REQUESTED** [review]

**Review by nahuelverdugo — APPROVED** [review]

**Pull request merged**